person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @person_rentals.push(rental)
+    rental.person_rental = self
+  end
+
```

Comment:
- Hey, let's kindly make some improvements to your implementation by adding a date and book to the parameter and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
Done!